src/main/java/edu/kit/ipd/crowdcontrol/objectservice/moneytransfer/MoneyTransferManager.java

```diff
@@ -297,26 +319,44 @@ private void payWorker(WorkerRecord worker, List<GiftCodeRecord> giftCodes) thro
     }
 
     /**
-     * Sends a notification about problems during payment to the administrator.
+     * Sends a notification with information about payment to the administrator.
      *
      * @param message the message to send
      * @throws MoneyTransferException gets thrown, if an error occurred
      */
-    private void sendNotification(String message) throws MoneyTransferException {
-        String subject = "Payment Notification";
-        StringBuilder mail;
+    private void sendNotification(NotificationLevel level, String message) throws MoneyTransferException {
+        Properties properties = new Properties();
         try {
-            mail = new StringBuilder(loadMessage("src/main/resources/moneytransfer/notificationMessage.txt"));
-        } catch (MoneyTransferException e) {
-            mail = new StringBuilder(e.toString());
+            InputStreamReader reader;
+            switch (level) {
+                case ERROR:
+                    reader = new InputStreamReader(Main.class.getResourceAsStream("/moneytransfer/errorNotification.properties"));
+                    break;
+                case ISSUE:
+                    reader = new InputStreamReader(Main.class.getResourceAsStream("/moneytransfer/issueNotification.properties"));
+                    break;
+                case GIFTCODES_ADDED:
+                    reader  = new InputStreamReader(Main.class.getResourceAsStream("/moneytransfer/giftcodesAddedNotification.properties"));
+                default:
+                    //cannot happen
+                    reader = null;
+            }
+            properties.load(reader);
+        } catch (IOException e) {
+            throw new MoneyTransferException("Error while loading properties for a notification.", e);
         }
+        String subject = properties.getProperty("subject");
+        String mail = Utils.loadFile(properties.getProperty("pathToMessage"));
 
-        mail = mail.append(message);
+        Map<String, String> map = new HashMap<>();
+        map.put("content", message);
+        mail = Template.apply(mail, map);
+        System.out.println(mail);
         try {
-            if (notificationText.length() != 0) {
-                LOGGER.trace("Started sending a notification about problems with submission of giftcodes.");
-                mailSender.sendMail(notificationMailAddress, subject, mail.toString());
-                LOGGER.trace("Completed sending a notification about problems with submission of giftcodes.");
+            if (mail.length() != 0) {
+                LOGGER.trace("Started sending a notification about " + properties.getProperty("loggerMessage") + ".");
+                mailSender.sendMail(notificationMailAddress, subject, mail);
+                LOGGER.trace("Completed sending a notification about " + properties.getProperty("loggerMessage") + ".");
```

Comment:
I guess its interesting to know, if an error occurrs with moneytransfer if the user got a notification mail or not or if it was tried to send a mail. I think we can never log to much in moneytransfer.